libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -21,8 +21,9 @@ It has the same interface and semantics as [`cuda::std::barrier`], with the
 
 ## Barrier Operations
 
-| [`cuda::barrier::init`]                 | Initialize a `cuda::barrier`. `(friend function)`                 |
-| [`cuda::device::barrier_native_handle`] | Get the native handle to a `cuda::barrier`. `(function template)` |
+| [`cuda::barrier::init`]                 | Initialize a `cuda::barrier`. `(friend function)`                       |
+| [`cuda::device::barrier_native_handle`] | Get the native handle to a `cuda::barrier`. `(function template)`       |
+| [`cuda::device::arrive_tx`]             | Arrive on a `cuda::barrier` with transaction count update. `(function)` |
```

Comment:
This is the first PR that turns barrier from a single expected count to expected count + transaction count. 
That is an extension over std::barrier that needs to be documented in the high-level documentation.

Reply:
I added a description of transaction count support in `cuda::barrier` located in shared memory in commit https://github.com/NVIDIA/cccl/pull/358/commits/c88f15b2776966bf2063e5cad8fd8b104c1c0c76